client/src/components/Jobs/Jobs.jsx

```diff
@@ -112,9 +137,10 @@ function JobsView({ showDetails, showFancyStatus, rowProps }) {
       headerStyle: { textAlign: 'left' },
       accessor: '_links.archiveData.href',
       Cell: (el) => {
-        const outputPath =
-          el.row.original.outputLocation || getOutputPathFromHref(el.value);
-        return outputPath && !hideDataFiles ? (
+        console.log(el);
+        const outputPath = '//data/awesome';
+        // el.row.original.outputLocation || getOutputPathFromHref(el.value);
```

Comment:
looks like `outputLocation` is defined in `jobs.reducers.js`. We can define it as `archiveSystemId` + `archiveSystemDir`. And we at least have `archiveSystemId` in a `getJobList()` call